src/modules/auth/resources/ResourcesListContainer.js

```diff
@@ -19,6 +19,7 @@ function ResourcesListContainer(props) {
 	const ref = useRef(null);
 	const { t } = useTranslation();
 
+	const deleteResource = "seacat:resource:edit";
```

Comment:
@petrKavulok do you really need it? take a look a line below, just use the resource on line 23, its the same